### PR TITLE
Update README with Bitcoin

### PR DESCRIPTION
**Pull request by markmhendrickson**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/hirosystems/stacks-wallet-web/actions/runs/4405599109).

### CODE REVIEW

**Review by fbwoolf — APPROVED** [review]
You just need to fix the commit msg to be something like `refactor: readme for bitcoin` so will pass test ...or I can?

File: README.md [hunk 1]
```diff
@@ -5,17 +5,17 @@
 [![coverage](https://raw.githubusercontent.com/hirosystems/stacks-wallet-web/gh-pages/badge.svg)](https://hirosystems.github.io/stacks-wallet-web/)
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
-Hiro Wallet is a web extension for managing your digital assets, and connecting to apps built on the Stacks blockchain.
+Hiro Wallet is the most popular and trusted wallet for apps built on Bitcoin. Connect to apps and manage assets secured by Bitcoin and BTC L2s with the most popular, battle-tested wallet for the Stacks blockchain.
```
Comment (kyranjamie) [inline README.md]: Isn't it better to say, even if repetitive, Bitcoin L2. BTC refers specifically to units of the currency itself. 

We shouldn't repeat the word "popular" twice either. I would omit the second usage entirely.

Comment (markmhendrickson) [reply README.md]: Agreed re these changes

**Comment by markmhendrickson** [comment]
> You just need to fix the commit msg to be something like refactor: readme for bitcoin so will pass test ...or I can?

@fbwoolf would you mind? 😬

**Comment by alter-eggo** [comment]
@markmhx changed commit message and fixed Kyran's comments

**Pull request merged**